docs/includes/net-standard-table.md

```diff
@@ -0,0 +1,26 @@
+| .NET Standard              | [1.0] | [1.1]  | [1.2] | [1.3] | [1.4] | [1.5]  | [1.6]  | [2.0] |
+|----------------------------|-------|--------|-------|-------|-------|--------|--------|-------|
+| .NET Core                  | 1.0   | 1.0    | 1.0   | 1.0   | 1.0   | 1.0    | 1.0    | 2.0   |
+| .NET Framework             | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.2* | vNext* | 4.6.1 |
```

Comment:
In the net-standard-table use one row for each tooling version for .net framework.

---

| .NET Standard                                            | 1.0   | 1.1    | 1.2   | 1.3   | 1.4    | 1.5     | 1.6       | 2.0     |
|----------------------------------------------|------|-------|------|------|-------|-------|---------|-------|
| .NET Framework (with tooling 1.0)            | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.2  | vNext   | 4.6.1 |
| .NET Framework (with tooling 2.0)            | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.1   | 4.6.1    | 4.6.1 |

---

This makes it much more clearer than before.
It makes clear for me at least :-)

Any thoughts?